src/main/java/app/linguistai/bmvp/service/AccountService.java

```diff
@@ -97,6 +86,17 @@ public RLoginUser login(QUserLogin user) throws Exception {
         }
     }
 
+    public void loginWithValidToken(String email) throws Exception {
+        // This method is only reached if the user already has a valid token
+        try {
+            initiateUserSession(email);
```

Comment:
Although it might seem redundant, adding `this.` prefix to in-service private function calls can improve the readability and maintainability of the code

Reply:
Added